docs/developer-docs/integrations/sns/launching/launch-summary.md

```diff
@@ -0,0 +1,332 @@
+| Swap account on the SNS Ledger                    | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+
+- #### Step 9: NNS DAO accepts or rejects the proposal
+
+  This is the **last of three** proposals that need to successfully pass for the process to continue. 
+
+  When voting on this proposal, the NNS voters can verify the parameters in the already
+  existing SNS canisters as well as the swap parameters that are set in the proposal.
+   
+  * If the NNS proposal is adopted, the swap is started after a specified delay. 
+  * If the NNS proposal is rejected, the SNS launch is aborted and the dapp’s control is handed
+  back to the original developers of the dapp.
+
+  :::info
+  Once this NNS proposal passes, the rest of the process should complete.
+  :::
+
+    If successful, at the end of stage, we have:
+
+|  Decentralization swap state                      | OPEN                                    |  
+|-------------------                                | ----------------                        |
+
+
+|  Objects in an app subnet                                      | State                    |  Controlled by |
+|-------------------                                | ----------------                      | ----------------|
+| A dapp                                            | Operational                           | ~~Dapp developer principal~~ Newly-created SNS canisters | 
+
+|  Objects in the SNS subnet                                       | State                  |  Controlled by |
+|-------------------                                | ----------------                      | ----------------|
+| ~~A principal that can deploy to SNS subnet~~     | Revoked because it is 1-time use      | NA                        |
+| Newly-created SNS canisters on the SNS subnet     | Pre-decentralization swap mode        | Initial developer neurons |  
+| Initial developer neurons                         | Pre-decentralization swap mode        | Dapp developer principal  |
+| Treasury account on the SNS Ledger                | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+| Swap account on the SNS Ledger                    | Pre-decentralization swap mode        | Newly-created SNS canisters on the SNS subnet   |
+
+
+- #### Step 10: SNS participants participate in the decentralization swap.
+
+  When the swap starts, the swap canister holds the number
+  of SNS tokens that were specified. End users can
+  participate in the decentralization swap by transferring ICP tokens to the
+  swap canister.
+
+  
+
+- #### Step 11: SNS canisters become SNS DAO.{#SNS-launch-step-genesis}
+  
+  When the decentralization swap ends, it is first established whether
+  it was successful, e.g., enough ICP have been collected. If the swap was successful,
```

Comment:
we give here an example. do we somewhere list all the details?
Should there maybe be a dedicated "SNS swap" page with this more detailed info, possibly also including as an intro the intro here?

Reply:
maybe? good idea